src/components/atoms/GroupCard.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Image from 'next/image';
+
+interface Props{
+    labelCard: string,
+    pathImage: string
+}
+
+const GroupCard = ({labelCard, pathImage}: Props) => {
+  return (
+    <div className='flex flex-col justify-center gap-y-[14px]'>
+        {
+          pathImage===""?
+          <div className='px-[53px] pb-[106px] bg-gray-opaque rounded-xl '/>          
```

Comment:
avoid using fixed sizes like p-[53px]. Use rem units, instead. For instance, p-2 or p-4